src/tests/execute-wp-cli.test.ts

```diff
@@ -31,6 +31,10 @@ describe( 'executeWPCli', () => {
 	} );
 
 	it( 'should return error if wp-cli command does not exist', async () => {
+		const originalConsoleError = console.error;
+		const originalConsoleWarn = console.warn;
+		console.error = jest.fn();
+		console.warn = jest.fn();
```

Comment:
As noted in the above test description, we expect an error to occur. Silencing it for test output reduces noise.